### PR TITLE
Value propositions: hide parts for dropshipped products

### PR DESCRIPTION
**Pull request by danielbeardsley**
Hide the "shipping" parts of our value propositions for products that are not shipped by us.

Originally: #965 but simplified here

Closes https://github.com/iFixit/ifixit/issues/45298

CC @dhmacs

### CODE REVIEW

File: frontend/templates/product/sections/ServiceValuePropositionSection/index.tsx [hunk 1]
```diff
@@ -12,10 +13,15 @@ import {
    faShieldCheck,
 } from '@fortawesome/pro-solid-svg-icons';
 import { FaIcon, FaIconProps } from '@ifixit/icons';
+import { useSelectedVariant } from '../../hooks/useSelectedVariant';
```
Comment (dhmacs) [inline frontend/templates/product/sections/ServiceValuePropositionSection/index.tsx:16]: This is not being used

```suggestion
```

**Review by dhmacs — APPROVED** [review]
lgtm! I've suggested a minor change, but it's fine to merge as is 👍 

CR ⚡

**Review by dhmacs — APPROVED** [review]
CR ⚡

**Comment by deltuh-vee** [comment]
dev_block ✊ 
I still see the value proposition sections for dropshipped items. As an example, I chose Asus C302CA Laptop Battery(IF335-093-1). 
image
image

**Comment by danielbeardsley** [comment]
Good catch @deltuh-vee !  Turns out the bug wasn't in this code and this code works when the metafield is edited to have the right stuff:

image

image

un_dev_block :+1: 

QA :+1: 

I'll open up a pull on our php app with the fix at the source of the data.

**Pull request merged**